webpack/stories/index.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+/* eslint-disable no-unused-vars */
+import { storiesOf, action, linkTo, addDecorator } from '@kadira/storybook';
+require('../assets/javascripts/bundle');
+require('../../app/assets/javascripts/application');
+import StatisticsChartsList from
+'../assets/javascripts/react_app/components/charts/StatisticsChartsList';
+import StatisticsChartBox from
+'../assets/javascripts/react_app/components/charts/StatisticsChartBox';
+import PowerStatus from
+'../assets/javascripts/react_app/components/hosts/PowerStatus';
+
+addDecorator((story) => (
+  <div className="ca" style={{textAlign: 'center'}}>
+    {story()}
+    <div id="targetChart"></div>
+  </div>
+));
+
+storiesOf('Statistics', module)
+  .add('Loading', () => (
+    <StatisticsChartBox
+      config={{data: {columns: [] }}}
+      noDataMsg={'No data here'}
+      title="Title"
+      status="PENDING" />
+  ))
+    .add('Without Data', () => (
+    <StatisticsChartBox
+      config={{data: {columns: [] }}}
+      noDataMsg={'No data here'}
+      title="Title"
+      status="RESOLVED" />
+  ))
+    .add('With Error', () => (
+    <StatisticsChartBox
+      config={{data: {columns: [] }}}
+      title="Title"
+      noDataMsg={'No data here'}
+      errorText="Ooops"
+      status="ERROR" />
+  ))
+  .add('With data', () => (
+    <StatisticsChartBox
+      config={{data: {columns: [1, 2]}}}
+      modalConfig={{}}
+      noDataMsg={'No data here'}
+      id="target"
+      status="RESOLVED"
+    />
+  )
+);
+
+storiesOf('Power Status', module)
+  .add('Loading', () => (
```

Comment:
@tbrisker I removed the other components and replace them with power status instead.

@gailsteiger from some reason, the imported css of the PowerStatus.js is not visible in storybook, i suspect something in webpack, but I was not able to identify it, could you please have a look?